ydb/core/protos/pqconfig.proto

```diff
@@ -306,6 +315,7 @@ message TPQTabletConfig {
         repeated string Codecs = 2;
     }
     repeated TCodecs ConsumerCodecs = 16;
+    repeated EConsumerScalingSupport ConsumerScalingSupport = 37;
```

Comment:
Can we make user friendly config with repeated TConsumer consumers ; TConsumer {name, readFromTs, formatVersion ...}? If format there is old format of config then treat that threre is no Scalingsupport.